src/stats.rs

```diff
@@ -581,7 +580,7 @@ impl<T: StatisticsLogFormatter> StatsConfigBuilder<T> {
     /// Set the Tokio reactor core to use for the logging of the statistics.
     // LCOV_EXCL_START No testing for this directly - simple code and a pain to
     // create tokio setups in UT.
-    pub fn with_core(mut self, handle: Handle) -> Self {
+    pub fn with_runtime(mut self, handle: Handle) -> Self {
```

Comment:
This is now technically a breaking change - I wonder if keeping the old name (at least as a deprecated alias) would be easier for anyone upgrading?  Breaking changes here are a pain because so many of our internal libraries have dependencies on it...

Reply:
I think it was a breaking change anyway, because it's a different type of `Handle` now.  

So in that case, I'd think it's actually better to make this conspicuous.

Reply:
Ah - yes, true.  Fine.